### PR TITLE
host-ovs: add missing network_type and fix pgrep

### PR DESCRIPTION
**Pull request by rbbratta**

### CODE REVIEW

**Comment by rbbratta** [comment]
fixes OCP-25707

File: features/step_definitions/networking.rb [hunk 1]
```diff
@@ -744,6 +742,8 @@
     end
   end
   unless host_ovs
+    network_operator = BushSlicer::NetworkOperator.new(name: "cluster", env: env)
```
Comment (pruan-rht) [inline features/step_definitions/networking.rb:745]: @rbbratta probably can combine these two lines into one?

Comment (rbbratta) [reply features/step_definitions/networking.rb:745]: I'd rather replace all the instances at once, but this current PR fixes current test failures, can we merge while I refactor `network_type`?

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**